APISpecification.yaml

```diff
@@ -277,6 +292,27 @@ components:
           type: array
           items:
             $ref: '#/components/schemas/Variant'
+    VepSubmissionStatusResponse:
+      type: object
+      required:
+        - submission_id
+        - status
+      properties:
+        submission_id:
+          type: string
+        status:
+          type: string
+          description: >
+            The possible statuses for a VEP submission are based on the list of documented statuses of Seqera workflows
```

Comment:
I'd shorten this decsription to omit implementation details/suggestions/documentation URLs (can be found from linked PR and JIRA tickets).